Source: partially based on debian patch
Upstream: No
Reason: set default terminal to lxterminal
--- lxde-common-0.3.2.1/lxpanel/panel.in	2009-09-28 02:54:29.618324398 +0800
+++ lxde-common-0.3.2.1/lxpanel/panel.in~	2009-09-28 02:51:54.725334092 +0800
@@ -53,7 +53,7 @@
             id=pcmanfm.desktop
         }
         Button {
-            id=gnome-terminal.desktop
+            id=lxterminal.desktop
         }
         Button {
             id=firefox.desktop
